docs/sources/dashboards/dashboard-public/index.md

```diff
@@ -223,6 +215,7 @@ If you've confirmed one of these data sources work with public dashboards, let u
         <li>GitHub</li>
         <li>Google BigQuery</li>
         <li>Grafana for YNAB</li>
+        <li>Graphite</li>
```

Comment:
Should we introduce now a new state "not compatible"?  In that case, Graphite should be part of that list with Cloudwatch.

Reply:
mmm yes, makes sense. Currently, we know and we're sure both are not compatible